src/nbody.effekt

```diff
@@ -0,0 +1,205 @@
+import mutable/array
+import src/benchmark
+
+record Body(
+  x: Double,
+  y: Double,
+  z: Double,
+  vx: Double,
+  vy: Double,
+  vz: Double,
+  mass: Double
+)
+
+def makeBody( 
+  x: Double,
+  y: Double,
+  z: Double,
+  vx: Double,
+  vy: Double,
+  vz: Double,
+  mass: Double): Body = {
+  val PI = 3.141592653589793;
+  val DAYS_PER_YER = 365.24;
+  val SOLAR_MASS = 4.0 * PI * PI;
+  Body(x,y,z, vx * DAYS_PER_YER, vy * DAYS_PER_YER, vz * DAYS_PER_YER, mass * SOLAR_MASS)
+}
+
+def nbody(problemSize: Int) = {
+  val PI = 3.141592653589793
+  val SOLAR_MASS = 4.0 * PI * PI
+  val DAYS_PER_YER = 365.24;
+
+  def offsetMomentum(px: Double, py: Double, pz: Double, body: Body) = {
+    Body(
+      body.x, body.y, body.z,
+      0.0 -(px / SOLAR_MASS),
```

Comment:
We should add a definition `def neg(x: Double): Double` to the standard library. Likewise for the bitshift operations you've added in another benchmark. Perhaps you could make a pull request on the effekt repo?